README.md

```diff
@@ -122,7 +122,11 @@ Make sure to complete these requirements:
 
 1️⃣ The FalkorDB repository: `git clone --recurse-submodules -j8 https://github.com/FalkorDB/FalkorDB.git`
 
->On Ubuntu Linux, run: `apt-get install build-essential cmake m4 automake peg libtool autoconf python3 python3-pip`
+>On Ubuntu Linux, run: `apt-get install build-essential cmake m4 automake peg libtool autoconf python3 python3-pip libssl-dev`
+>
+>when your python3 --version says < 3.7 write:
```

Comment:
In case your python version is older than 3.7, you'll have to `pip install dataclasses`